kmq.js

```diff
@@ -36,6 +36,25 @@ client.on("message", (message) => {
     }
 });
 
+client.on("voiceStateUpdate", (oldMember, newMember) => {
+    let oldUserChannel = oldMember.voiceChannel;
+    let newUserChannel = newMember.voiceChannel;
+    if (newUserChannel === undefined) {
```

Comment:
Why is the voicestate undefined? I'm struggling to understand how this works, especially given that the bot is still in the voice channel.